nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -222,6 +223,26 @@ def backup(self, backup_dist=0.15, backup_speed=0.025, time_allowance=10):
         self.result_future = self.goal_handle.get_result_async()
         return True
 
+    def assistedTeleop(self, time_allowance=0):
```

Comment:
What default did we set in the cpp files? I don't think the infinite period is a good default. Maybe 30 seconds?

Reply:
Switched, there is no default in the cpp file :). So I just set it to 30 sec.